commit 711bea86d3c37e89b531b2cf1b530059c8561e0f (refs/changes/71/2071/4)
Author: Josh Wilsdon <jwilsdon@joyent.com>
Date:   2017-06-09T23:28:32+00:00 (2 years, 4 months ago)
    
    VOLAPI-50 Add ListVolumeSizes endpoint (disambiguate endpoints)
    Reviewed by: Julien Gilli <julien.gilli@joyent.com>
    Approved by: Julien Gilli <julien.gilli@joyent.com>

diff --git a/lib/endpoints/volumes.js b/lib/endpoints/volumes.js
index 0abbb77..469bc69 100644
--- a/lib/endpoints/volumes.js
+++ b/lib/endpoints/volumes.js
@@ -1490,7 +1490,7 @@ function mount(config, server, applicationState) {
         }));
 
     server.get({
-        path: '/volumes/sizes',
+        path: '/volumesizes',
         name: 'ListVolumeSizes',
         version: '1.0.0'
     }, restify.queryParser(), validateListVolumeSizes, listVolumeSizes,
diff --git a/test/integration/nfs-shared-volumes-list-volume-sizes.test.js b/test/integration/nfs-shared-volumes-list-volume-sizes.test.js
index 33e0b88..4b11288 100644
--- a/test/integration/nfs-shared-volumes-list-volume-sizes.test.js
+++ b/test/integration/nfs-shared-volumes-list-volume-sizes.test.js
@@ -32,7 +32,7 @@ test('setup', function (tt) {
 });
 
 test('listVolumeSizes', function (tt) {
-    tt.test(' GET /volumes/sizes should return array of sizes', function (t) {
+    tt.test(' GET /volumesizes should return array of sizes', function (t) {
         CLIENTS.volapi.listVolumeSizes({type: 'tritonnfs'},
             function onListVolumeSizes(err, volumeSizes) {
                 var idx = 0;
